src/app/viewing/messages/messages.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { ViewingService } from '../../core/viewing.service';
+
+@Component({
+  selector: 'app-messages',
+  templateUrl: './messages.component.html',
+  styleUrls: ['./messages.component.css']
+})
+
+export class MessagesComponent implements OnInit {
+  userMsgData;
+  constructor(public viewingService: ViewingService) {}
+
+  ngOnInit(): void {
+    this.userMsgData = this.viewingService.userMsgData;
+  }
+
+  toggleViewStyle() {
+    this.viewingService.isCardView = this.viewingService.isCardView === false ? true : false;
```

Comment:
Simplify this expression
```suggestion
    this.viewingService.isCardView = this.viewingService.isCardView === false;
```